src/components/icon/icons/Apple.jsx

```diff
@@ -0,0 +1,8 @@
+const AppleIcon = () => (
+	<path
+		d="M19.5577 12.6827C19.5465 10.7162 20.4726 9.23204 22.3472 8.13898C21.2984 6.69765 19.7139 5.90465 17.6217 5.74926C15.6411 5.59923 13.4764 6.85839 12.6841 6.85839C11.8472 6.85839 9.928 5.80284 8.42163 5.80284C5.30845 5.85106 2 8.1872 2 12.9399C2 14.3437 2.2678 15.7957 2.8034 17.2906C3.51753 19.2571 6.09511 24.0794 8.78427 23.999C10.1902 23.9669 11.1833 23.0399 13.0133 23.0399C14.7875 23.0399 15.708 23.999 17.2758 23.999C19.9872 23.9615 22.3193 19.5786 23 17.6068C19.3624 15.9618 19.5577 12.7845 19.5577 12.6827ZM16.3998 3.88464C17.923 2.14861 17.7835 0.56796 17.7388 0C16.3943 0.0750136 14.8377 0.878731 13.9506 1.86998C12.9742 2.93089 12.3996 4.24363 12.5223 5.72247C13.9785 5.82963 15.3063 5.11164 16.3998 3.88464Z"
+		fill="black"
```

Comment:
The fill color should not be set here. User's would not be able to style the icon color if this is hard coded. We have future work slated to make the fill color a prop that can be passed in, but we should leave it off for now.